binr/rahash2/rahash2.c

```diff
@@ -356,6 +364,10 @@ static int encrypt_or_decrypt(const char *algo, int direction, const char *hashs
 				ut8 *result = r_crypto_get_output (cry, &result_size);
 				if (result) {
 					write (1, result, result_size);
+					// Add linebreak by default
+					if (linebreak) {
+						write (1,  "\n", 1);
```

Comment:
adding a line break by default in the output of an encrypted or decrypted output will break the contents and make it unusable